packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/terminateEmail/email.routing.js

```diff
@@ -11,13 +11,25 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       email: /* @ngInject */ (BillingAutoRenew, name) =>
         BillingAutoRenew.getEmailInfos(name),
-      isHosting: /* @ngInject */ (email) => ['hosting'].includes(email.offer),
+      isHosting: /* @ngInject */ (email) => /hosting/.test(email.offer),
       goBack: /* @ngInject */ (goToAutorenew) => goToAutorenew,
       name: /* @ngInject */ ($transition$) => $transition$.params().name,
       serviceId: /* @ngInject */ ($transition$) =>
         $transition$.params().serviceId,
-      terminateEmail: /* @ngInject */ (BillingAutoRenew, serviceId) => () =>
-        BillingAutoRenew.terminateEmail(serviceId),
+      terminateEmail: /* @ngInject */ (
+        $q,
+        BillingAutoRenew,
+        email,
+        isHosting,
+      ) => () => {
+        if (isHosting) {
+          window.location.href = `/web/#/configuration/hosting/${encodeURIComponent(
```

Comment:
Maybe this could be stored as a constant instead ?

Reply:
You're right it could be enhanced in that way.

Since this ticket has been validated I will merge it. Feel free to open an Issue regarding your suggestion in order to be able to process it.

Thanks!